run_test.sh

```diff
@@ -427,7 +491,7 @@ EOF
 echo 'Removing /usr/src/dkms_test-1.0'
 rm -r /usr/src/dkms_test-1.0
 
-echo 'Checking that the environment is clean again'
+echo 'Checking that the environment is clean'
```

Comment:
Ideally cosmetics like these should be separate change, from functionality like the rest of the patch.